packages/flutter_availability/lib/src/ui/screens/template_day_modification.dart

```diff
@@ -66,97 +67,69 @@ class _DayTemplateModificationScreenState
-      context,
-      canSave ? onSavePressed : null,
-      Text(translations.saveButton),
-    );
-
     var deleteButton = options.bigTextButtonBuilder(
       context,
       onDeletePressed,
       Text(translations.deleteTemplateButton),
     );
 
-    var title = Center(
-      child: Text(
-        translations.dayTemplateTitle,
-        style: theme.textTheme.displaySmall,
-      ),
-    );
-
-    var templateTitleSection = TemplateNameInput(
-      initialValue: _viewModel.name,
-      onNameChanged: (name) {
-        setState(() {
-          _viewModel = _viewModel.copyWith(name: name);
-        });
-      },
-    );
+    void onNameChanged(String name) {
+      setState(() {
+        _viewModel = _viewModel.copyWith(name: name);
+      });
+    }
 
-    var colorSection = TemplateColorSelection(
-      selectedColor: _viewModel.color,
-      // TODO(Joey): Extract this
-      onColorSelected: (color) {
-        setState(() {
-          _viewModel = _viewModel.copyWith(color: color);
-        });
-      },
-    );
+    void onColorSelected(int? color) {
+      setState(() {
+        _viewModel = _viewModel.copyWith(color: color);
+      });
+    }
 
-    var availabilitySection = TemplateTimeAndBreakSection(
-      dayData: _viewModel.data,
-      onDayDataChanged: (data) {
-        setState(() {
-          _viewModel = _viewModel.copyWith(data: data);
-        });
-      },
-    );
+    void onDayDataChanged(DayTemplateDataViewModel data) {
+      setState(() {
+        _viewModel = _viewModel.copyWith(data: data);
+      });
+    }
 
-    var body = CustomScrollView(
-      slivers: [
-        SliverPadding(
-          padding: EdgeInsets.symmetric(horizontal: spacing.sidePadding),
-          sliver: SliverList.list(
-            children: [
-              const SizedBox(height: 40),
-              title,
-              const SizedBox(height: 24),
-              templateTitleSection,
-              const SizedBox(height: 24),
-              availabilitySection,
-              const SizedBox(height: 24),
-              colorSection,
-              const SizedBox(height: 32),
-            ],
-          ),
-        ),
-        SliverFillRemaining(
-          fillOverscroll: false,
-          hasScrollBody: false,
-          child: Padding(
-            padding: EdgeInsets.symmetric(
-              horizontal: spacing.sidePadding,
-            ).copyWith(
-              bottom: spacing.bottomButtonPadding,
-            ),
-            child: Align(
-              alignment: Alignment.bottomCenter,
-              child: Column(
-                children: [
-                  saveButton,
-                  if (widget.template != null) ...[
-                    const SizedBox(height: 8),
-                    deleteButton,
-                  ],
-                ],
-              ),
+    return options.baseScreenBuilder(
+      context,
+      widget.onExit,
+      BasePage(
+        body: [
```

Comment:
extract this

Reply:
What would be the point of that? Most of this has already been extracted, I see no use at all in moving this to a variable.